src/System.Runtime.Extensions/tests/System/AppDomainTests.netfx.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Tests
+{
+    public partial class AppDomainTests
+    {
+        [Fact]
+        public void GetSetupInformation()
+        {
+            Assert.Equal(AppContext.BaseDirectory, AppDomain.CurrentDomain.SetupInformation.ApplicationBase);
+            // AppContext.TargetFrameworkName is not defined in .NetFramework 4.6.2 or lower
+            Assert.Contains(".NETFramework", AppDomain.CurrentDomain.SetupInformation.TargetFrameworkName);
+        }
```

Comment:
Why have two different versions of the test in two different AppDomainTests.*.cs files, rather than just testing something different at runtime based on, for example, using PlatformDetection in one test?

Reply:
Agree. This split does not look good.

Reply:
we can rewrite it as after #23063 has been resolved. 
```C#
[Fact]
public void GetSetupInformation()
{
    Assert.Equal(AppContext.BaseDirectory, AppDomain.CurrentDomain.SetupInformation.ApplicationBase);
    if (PlatformDetection.isFullFramework)
    {   
     // AppContext.TargetFrameworkName is not defined in .NetFramework 4.6.2 or lower
         Assert.Contains(".NETFramework", AppDomain.CurrentDomain.SetupInformation.TargetFrameworkName);
   }
   else
   {
       Assert.Contains(".NETCore", AppDomain.CurrentDomain.SetupInformation.TargetFrameworkName);
 
   }
}
```
i can open a pr to resolve #23063 and then merge this PR ?

Reply:
I would rather we not introduce two files now only to combine them later; it's unnecessary churn. Just disable the test on a given platform if you need to as we often do elsewhere, linked to that issue/PR.